.travis-addon.yml.example

```diff
@@ -3,12 +3,8 @@ python:
   - "3.6"
 
 install:
-  - pip install Pillow requests
-
-before_install:
-  - git submodule update --init --recursive --remote
+  - pip install Pillow requests kodi-addon-checker
 
 # command to run our tests
 script: 
-  - cd .tests 
-  - python check_addon.py
\ No newline at end of file
+  - kodi-addon-checker .
```

Comment:
To check an entire repository the `.` is not necessary.
Directories passed as arguments are treated as plugins (The same behavior as the previous implementation)

This line should be:
```
- kodi-addon-checker
```